apps/theming/appinfo/routes.php

```diff
@@ -39,6 +39,11 @@
 			'url' => '/ajax/undoChanges',
 			'verb' => 'POST'
 		],
+		[
+			'name' => 'Theming#undoAll',
+			'url' => '/ajax/undoAllChanges',
+			'verb' => 'POST'
```

Comment:
```suggestion
			'verb' => 'PUT'
```
Seems more fitting https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/PUT ?

Reply:
I'm following the existing UNDO api method :)

PUT is good if the url targets a ressource.
Here it's a global action, POST is acceptable then I would say, but I digress :grin: